### DISCUSSION TITLE
Temperature on Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by ghost** [netdata/netdata#13038] [Q&A] [answered]
Trying to enable temp chart but I keep getting the below error :(

<img width="583" alt="Screen Shot 2022-05-30 at 8 39 23 PM" src="https://user-images.githubusercontent.com/99628102/171076250-52c3781b-4ffd-4474-9294-7c693ed616d3.png">

As you can see my cd path is correct?!
<img width="458" alt="Screen Shot 2022-05-30 at 8 41 50 PM" src="https://user-images.githubusercontent.com/99628102/171076306-8f766837-94e6-4c1c-adfe-a649c1b81c21.png">

**Comment by MrZammler** [comment, marked as answer]
Hi @checked-wobbles-0p !

The `edit-config` script should be in yout /opt/netdata/etc/netdata folder.

Any custom changes you want to make to scripts should live there, and will take precedence over the stock scripts in `/usr/lib`.
